pkg/operator/ceph/controller/spec.go

```diff
@@ -653,8 +655,23 @@ func PrivilegedContext(runAsRoot bool) *v1.SecurityContext {
 	return sec
 }
 
-// LogCollectorContainer runs a cron job to rotate logs
+// LogCollectorContainer rotate logs
 func LogCollectorContainer(daemonID, ns string, c cephv1.ClusterSpec) *v1.Container {
+
+	maxLogSize := "500M"
+	if c.LogCollector.MaxLogSize != nil {
+		maxLogSize = fmt.Sprintf("%dM", c.LogCollector.MaxLogSize.Value()/1000/1000)
```

Comment:
What about checking for the minimum? We could just change it to 1M if it ends up at 0M?

Reply:
That might defy user expectations if they want to set something like 500k, but I think it's also reasonable.

Reply:
>Who is going to have that low of a limit anyway?
also, we have added a note the minimum should be 1M but added the check to be on the safe side


added the check for 0M and then overrode it to 1M with the log message.